### PR TITLE
Fix linter

### PR DESCRIPTION
**Pull request by adibrastegarnia**
@kuujo 

there are some linter errors and the name of interfaces should be changed to avoid adding "helm" to the beginning of them. Any suggestion? The linter itself suggests to remove helm at the beginning of interface/struct names or we should rename the package.

### CODE REVIEW

**Comment by kuujo** [comment]
@adibrastegarnia the use of the `Helm` prefix was intentional. It's an API design decision because those types would have conflicted with the package level functions that are the API for creating them. For example:

```go
helm.Chart("foo").Release("bar").Install(true)
```

If the chart type is named `helm.Chart` then the `helm.Chart` function cannot exist. But since the types are never instantiated externally, we can maybe make them into interfaces like `ChartInterface` and `ReleaseInterface`. But I'm also just fine with ignoring the linter errors.

**Comment by adibrastegarnia** [comment]
@kuujo @ray-milkey ready to go

File: pkg/util/files/echo.go [hunk 1]
```diff
@@ -60,7 +61,7 @@ func (o *EchoOptions) To(filename string) *EchoOptions {
 	return o
 }
 
-// To configures the copy destination pod
+// On To configures the copy destination pod
```
Comment (kuujo) [inline pkg/util/files/echo.go:64]: This comment should be `On configures...`

Comment (adibrastegarnia) [reply pkg/util/files/echo.go]: Done

**Pull request merged**